database/migrations/sql/20181123_16_add_data_sync_log_table.sql

```diff
@@ -0,0 +1,13 @@
+/*
+ * Data sync table
+ * - keep record of data synced from old dbs
+ */
+
+ CREATE TABLE data_sync_log (
+  data_sync_log_id  SERIAL NOT NULL UNIQUE,
+  foreign_key       INT NOT NULL,
+  table_name        CITEXT NOT NULL,
```

Comment:
Any reason for `CITEXT` instead of `VARCHAR`?

Reply:
Nope, can change

Reply:
fixed in amend